packages/editor-ui/src/components/MainHeader/WorkflowDetails.spec.ts

```diff
@@ -0,0 +1,115 @@
+import WorkflowDetails from '@/components/MainHeader/WorkflowDetails.vue';
```

Comment:
I think we should agree on the convention about where to put these `.spec` files. In `design-system` we use `__tests__` folder while in `editor-ui` we are keeping them sometimes alongside the tested file (like this) and sometimes in `__tests__` folder

Reply:
We had a discussion about this during our last Front End meeting and decided we'd put them alongside the file being tested. I'll make a PR to move the rest of them accordingly.

Reply:
Ah, thanks for clearing that up 💡